resources/Microsoft.VSCode.Dsc/Microsoft.VSCode.Dsc.psm1

```diff
@@ -95,11 +95,32 @@ function Invoke-VSCode {
         [string]$Command
     )
 
-    try {
-        Invoke-Expression "& `"$VSCodeCLIPath`" $Command"
-    } catch {
-        throw ("Executing {0} with {$Command} failed." -f $VSCodeCLIPath)
+    $stdErrTempFile = "$env:TEMP\$((New-Guid).Guid)"
```

Comment:
I would say, use: [system.io.path]::GetTempPath() for MacOS and Linux support.

Reply:
Since right now we only have support for windows, we can hold off on this change for now. I want to get this checked in so that the bug is no longer an issue.